clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/deploy/ops/ResizeServiceAtomicOperation.java

```diff
@@ -69,26 +73,28 @@ private Service resizeService() {
 
     String serviceName = description.getServerGroupName();
     Integer desiredCount = description.getCapacity().getDesired();
+    String clusterName = containerInformationService.getClusterName(serviceName, description.getAccount(), description.getRegion());
 
     UpdateServiceRequest updateServiceRequest = new UpdateServiceRequest()
-      .withCluster(ecsClusterName)
+      .withCluster(clusterName)
```

Comment:
Why not call it ecsClusterName anymore?  it helps with confusions with spinnaker clusters, no?

Reply:
Done.